palm/utils.py

```diff
@@ -1,6 +1,49 @@
+from typing import Optional, Tuple
+import subprocess
+from sys import version_info
+
+
+class UnsupportedVersion(Exception):
+    pass
+
+
 def is_cmd_file(filename: str) -> bool:
     return filename.endswith(".py") and filename.startswith("cmd")
 
 
 def cmd_name_from_file(filename: str) -> str:
     return filename[4:-3]
+
+
+def run_on_host(
```

Comment:
Nit: It'd be nice to move this out of the utils.py "misc drawer" and into a better named module